Python/bigtable/read_bigtable.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import apache_beam as beam
+
+from apache_beam import Map
+from apache_beam.io.gcp.bigtableio import ReadFromBigtable
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+def run():
+    """
+    This Apache Beam pipeline retrieves data from Google Cloud Bigtable table.
```

Comment:
@Amar3tto i think we can use simple comments like in existing examples . Something like "This pipeline shows how to read from Cloud Bigtable"